build/tasks/build.js

```diff
@@ -3,24 +3,45 @@ var runSequence = require('run-sequence');
 var to5 = require('gulp-babel');
 var paths = require('../paths');
 var compilerOptions = require('../babel-options');
+var compilerTsOptions = require('../typescript-options');
 var assign = Object.assign || require('object.assign');
 var through2 = require('through2');
 var concat = require('gulp-concat');
 var insert = require('gulp-insert');
 var rename = require('gulp-rename');
 var tools = require('aurelia-tools');
-
+var ts = require('gulp-typescript');
+var gutil = require('gulp-util');
+var gulpIgnore = require('gulp-ignore');
+var merge = require('merge2');
 var jsName = paths.packageName + '.js';
+var compileToModules = ['es2015', 'commonjs', 'amd', 'system', 'native-modules'];
```

Comment:
@EisenbergEffect, I'm wondering if this build script (or part of it) could be moved to some other repo (aurelia-tools maybe?) so that other aurelia subprojects could reuse the same build script (for building typescript to different module formats) without duplicating the code in every repo where typescript is going to be taken into use.